System.Configuration.Abstractions/ConnectionStringsExtended.cs

```diff
@@ -63,5 +64,18 @@ private ConnectionStringSettings Intercept(ConnectionStringSettings rawSetting)
                 (current, interceptor) => interceptor.OnConnectionStringRetrieve(_appSettings, this, current));
             return rawSetting;
         }
+
+        IEnumerator IEnumerable.GetEnumerator()
+        {
+            return this.Raw.GetEnumerator();
+        }
+
+        public IEnumerator<ConnectionStringSettings> GetEnumerator()
+        {
+            for (int pos = 0; pos < Raw.Count; pos++)
+            {
+                yield return Raw[pos];
```

Comment:
Can we change this to 

```csharp
yield return Intercept(Raw[pos]);
```

To keep consistency with the rest of the API in ConnectionStringsExtended.